iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/ContentIntegrationTests.cpp

```diff
@@ -967,12 +967,12 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, DescriptorOverride_FiltersB
     BeGuid instanceGuid1 = BeGuid(true);
     IECInstancePtr instance1 = RulesEngineTestHelpers::InsertInstance(s_project->GetECDb(), *classA, [instanceGuid1](IECInstanceR instance) 
         {
-        instance.SetValue("GuidProp", ECValue((Byte*)&instanceGuid1, sizeof(BeGuid)));
+        instance.SetValue("GuidProp", ECValue((const Byte*)&instanceGuid1, sizeof(BeGuid)));

```

Comment:
Our convention is to use `Byte const*` rather than `const Byte*`.

Reply:
fixed